### ISSUE TITLE
Files named like .fuse_hidden00002b8600000001prevent deletion of folders after a video was played

### ISSUE THREAD

**Issue by druidasse** [xbmc/vfs.rar#159]
Hello,

Before 2 years ago or something like that I would:
     - play a video archived in multi rar files using your extension
     - delete the file from the hard drive when done
This caused no issue.

At some point, it became impossible to delete the folder the same way, I would have to do the following instead:
     - play a video archived in multi rar files using your extension
     - stop kodi
     - delete the file from the hard drive

I found out that this was due to files .fuse_hidden00002b8600000001 being locked bi kodi.bin:

$ sudo lsof /media/MyExtDisk/Video/MyMoviesFolder/.fuse_hidden00002b8600000001 
     COMMAND   PID   USER   FD   TYPE   DEVICE   SIZE/OFF   NODE   NAME
     kodi.bin         1353  kodi      67r   REG    8,17         33779228   206106  /media/MyExtDisk/Video/MyMoviesFolder/.fuse_hidden00002b8600000001
     kodi.bin         1353  kodi      75r   REG    8,17         33779228   206106  /media/MyExtDisk/Video/MyMoviesFolder/.fuse_hidden00002b8600000001

I did a lot of things to attempt fixing an issue at the OS level without any apparent luck. Then at some point this issue stopped. It seemed to have resolved at a time of an update of kodi/vfs.rar. I'm really not sure if that's really what resolved the issue or something else.

At that time the files were stored on a disk that was formated in exfat, the OS was Ubuntu 20.04, then upgraded to 22.04. 

CURRENT:
It was working fine until I just reinstalled it completely to Ubuntu 22.04 again + I reformatted the hard drive to NTFS instead. I have to stop kodi again if I want to be able to delete the folder of the movie I just watched.

After the reinstallation I also noticed that the system hard drive was getting full after I started to run full file scans to kodi's library of the drive where the movies are stored. Space would only be freed from the cache folder after I would close kodi. Shouldn't this be done immediately after the movie has been watched?

Any idea? Any setting I may be missing?

**Comment by druidasse** [comment]
Anyone?

**Comment by ibydos** [comment]
> Anyone?

Similar Problem here, see:
https://github.com/xbmc/vfs.rar/issues/98

Prevents me from updating to a current KODI version since ~3years as I use the last version without the bug 🤷‍♂️

**Comment by craigfergusson** [comment]
@AlwinEsch @notspiff @gade01 @akva2 @xhaggi @Rechi @ChrisJollyAU @weblate  @stefansaraev @KodiWeblate @ksooo @Sopor @xesarni @taxigps 
Finally made the move to Nexus and this addon still has the same deletion issue. Last working version I hade was 2.0.8. That's probably 4 years ago at least.
Please see also here: https://github.com/xbmc/vfs.rar/issues/98

Hope someone can finally fix it :(

**Comment by druidasse** [comment]
Indeed, it would be nice somebody fixes it. Not a big deal, but kind o annoying as in my setup I have to reboot the computer to work around this.
